src/assignment/components/AssignmentActions.tsx

```diff
@@ -33,9 +41,95 @@ const AssignmentActions: FunctionComponent<AssignmentActionsProps> = ({
 	overflow,
 	duplicate,
 	remove,
+	share,
 }) => {
 	const { tText } = useTranslation();
 	const [isOverflowDropdownOpen, setOverflowDropdownOpen] = useState<boolean>(false);
+	const [contributors, setContributors] = useState<Contributor[]>();
+
+	const fetchContributors = useCallback(async () => {
+		if (!share?.assignment?.id) {
+			return;
+		}
+		const response = await AssignmentService.fetchContributorsByAssignmentId(
+			share?.assignment?.id
+		);
+
+		setContributors(response as Contributor[]);
+	}, [share]);
+
+	useEffect(() => {
+		fetchContributors();
+	}, [share]);
```

Comment:
you don't need to repeat the [share] dependency from the useCallback function here. You can just make the useCallback function the dependency:
```suggestion
	}, [fetchContributors]);
```